pom.xml

```diff
@@ -1489,5 +1498,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <!-- oleg_nenashev: Cannot setup activation for multiple Java versions:
+           https://issues.apache.org/jira/browse/MNG-3328
+       -->
+      <id>java-level-11</id>
+      <activation>
+        <property>
+          <name>java.level</name>
```

Comment:
you can not activate properties like this.  It requires you run maven with `mvn -Djava.level=11` as it can only look at system properties - not properties defined in the `<properties>` section of the XML

Reply:
Not a big concern for me

Reply:
the problem is it doesn't work if you have a reactor with a mix of things - you need to move everything in one to java11 (the reactor may not be the parent and all the plugins could come from different repositories with different release cycles)